ideasbox/serveradmin/views.py

```diff
@@ -110,3 +114,81 @@ def backup(request):
 def battery(request):
     return render(request, 'serveradmin/battery.html',
                   {'batteries': batinfo.batteries()})
+
+
+unset_config_data = """scheme_current=None
+interface_current={interface}
+scheme_active=False
+""".format(interface=interface)
+
+def set_wifilist():
+    f = open('.runningconfig', 'r+')
```

Comment:
It is implemented in wifi lib to identify the connection status and related config data.